### ISSUE TITLE
Player:GetEyeTrace()

### ISSUE THREAD

**Issue by Etos7** [FPtje/DarkRP#2819]
[//]: # (DO NOT REMOVE THIS TEMPLATE.)
[//]: # (GITHUB ISSUES ARE **NOT** FOR HELP)
[//]: # (GO TO THE FORUMS INSTEAD: https://forum.darkrp.com/ )

### Description of the bug
[//]: #Player:GetEyeTrace().The entity does not work for money and tip jar ent. It prints out world/map ent, but not money ent. Tested on vanilla DarkRP singleplayer and multiplayer.  Also, ent owner is from prop protection is missing when looking on tip jar or money ent.

**Screenshot with ammo box working OK:**
https://prnt.sc/lmdo2k
`print(Player:GetEyeTrace().Entity) -> Entity [123][spawned_ammo]`

**Screenshot with jar working NOT OK:** 
https://prnt.sc/lmdq70
`print(Player:GetEyeTrace().Entity -> Entity [0][worldspawn]`

### How to make the bug happen
[//]: # print(Player:GetEyeTrace().Entity )
[//]: # Unknown

### Lua errors
[//]: # None

### Why the developer of DarkRP is responsible for this issue
[//]: # I'm using vanilla DarkRP, the issue is tested on my PC and my friends PC, both test on singleplayer and multiplayer. GetEyeTrace is standard GMod function and in my opinion, it should work with DarkRP money. It was working a few months ago I used it to print hud when the player was looking at money,

**Comment by FPtje** [comment]
`spawned_money` works for me, the tip jar does not.

**Comment by Etos7** [comment]
Found it! ` self:SetModelScale(1.5, 0)` is cousing this issue

**Comment by FPtje** [comment]
Actually, it's the combination `self:SetModelScale(1.5, 0)` and `self:Activate()`. I've tried several things. In the end I managed to have both a scaled model serverside **and** `:GetEyeTrace().Entity` working clientside, but with this caveat:

![gmod_2018-11-24_22-03-31](https://user-images.githubusercontent.com/1202014/48972929-f8ec4700-f034-11e8-95cd-1ac6c546f085.jpg)

Basically the physgun beam going haywire. I think it may be a bug or some odd behaviour that just isn't documented on `SetModel`'s [wiki page](http://wiki.garrysmod.com/page/Entity/SetModelScale). I don't think there's anything I can do about it. Sorry.
